src/rules/font-family-name-quotes/__tests__/index.js

```diff
@@ -96,6 +98,21 @@ testRule(rule, {
   }, {
     code: "a { font-family: -apple-system, 'BlinkMacSystemFont', 'Segoe UI', 'Roboto', sans-serif; }",
     message: messages.rejected("BlinkMacSystemFont"),
+  }, {
+    code: "a { font: italic 300 16px/30px Arial, serif; }",
+    message: messages.expected("Arial"),
+    line: 1,
+    column: 32,
+  }, {
+    code: "a { font: italic 1000 16px/30px Arial, serif; }",
+    message: messages.expected("1000 Arial"),
```

Comment:
> Would using `isNumbery()` avoid this?

@jeddy3 @davidtheclark  we need to do this to change that message if we want to.

i think we need to have a proper check `here`